src/bluesearch/mining/eval.py

```diff
@@ -423,34 +486,40 @@ def ner_report(iob_true, iob_pred, mode="entity", etypes_map=None, return_dict=F
 
 
 def ner_errors(
-    iob_true, iob_pred, tokens, mode="entity", etypes_map=None, return_dict=False
-):
+    iob_true: pd.Series,
+    iob_pred: pd.Series,
+    tokens: pd.Series,
+    mode: Optional[str] = "entity",
+    etypes_map: Optional[dict] = None,
+    return_dict: Optional[bool] = False,
```

Comment:
Same comment as above. I mean in `if` statements `None` will behave like a `False` and it passing `return_dict=None` is not going to error out but I guess the same argument  as above applies.

Reply:
Fixed all wrong usages of Optiona[...] in 1ecd77b.